// @colyseus/loadtest/build/package.mjs
var name = "@colyseus/loadtest";
var version = "0.14.6-alpha.3";
var description = "Utility tool for load testing Colyseus.";
var input = "./src/index.ts";
var main = "./build/index.js";
var module = "./build/index.mjs";
var typings = "./build/index.d.ts";
var bin = {
	"colyseus-loadtest": "./bin.js",
	"colyseus-loadtest-esm": "./bin.mjs"
};
var files = [
	"build",
	"bin.js",
	"bin.mjs",
	"tsconfig.json"
];
var repository = {
	type: "git",
	url: "git+ssh://git@github.com/colyseus/colyseus-loadtest.git"
};
var homepage = "https://colyseus.io/";
var keywords = [
	"colyseus",
	"load",
	"test",
	"load",
	"testing"
];
var author = "Endel Dreyer";
var license = "MIT";
var bugs = {
	url: "https://github.com/colyseus/colyseus-loadtest/issues"
};
var dependencies = {
	blessed: "^0.1.81",
	"colyseus.js": "^0.14.13",
	minimist: "^1.2.0",
	"ts-node": "^7.0.1",
	typescript: "^3.2.4",
	ws: "^6.1.4"
};
var devDependencies = {
	"@types/blessed": "^0.1.10"
};
var _package = {
	name: name,
	version: version,
	description: description,
	input: input,
	main: main,
	module: module,
	typings: typings,
	bin: bin,
	files: files,
	repository: repository,
	homepage: homepage,
	keywords: keywords,
	author: author,
	license: license,
	bugs: bugs,
	dependencies: dependencies,
	devDependencies: devDependencies
};

export default _package;
export { author, bin, bugs, dependencies, description, devDependencies, files, homepage, input, keywords, license, main, module, name, repository, typings, version };
